android-exoplayer/src/main/res/values/strings.xml

```diff
@@ -11,4 +11,12 @@
 
   <string name="unrecognized_media_format">Unrecognized media format</string>
 
+  <string name="error_drm_not_supported">Protected content not supported on API levels below 18</string>
+
+  <string name="error_drm_unknown">An unknown DRM error occurred</string>
+
+  <string name="error_drm_unsupported_scheme">This device does not support the required DRM scheme</string>
+
+  <string name="error_drm_session">An error with DRM session occurred</string>
```

Comment:
Would it be better to add the error keys in the messages? Then RN side can grep the error more easily, and no need to worry about the future changes of the error message sentences. :thinking: